xbmc/platform/posix/PlatformPosix.h

```diff
@@ -16,6 +16,7 @@ class CPlatformPosix : public CPlatform
 {
 public:
   bool Init() override;
+  bool IsConfigureAddonsAtStartupEnabled() override { return false; };
```

Comment:
No need for this. CPlatformPosix inherits the implementation from CPlatform.

Reply:
ok, then this was while i was playing with `bool XY override const { return false;};` ...declared override, but doesnt override.. build error. anyway, this one's boiling to a 2-liner i guess. cool